### PR TITLE
Adds documentation for pruning metrics

### PR DESCRIPTION
**Pull request by lucb-da**
CHANGELOG_BEGIN

CHANGELOG_END

### CODE REVIEW

File: docs/source/ops/common-metrics.rst [hunk 1]
```diff
@@ -4,13 +4,13 @@
 Common Metrics
 ##############
 
-The below sections contain the list of common metrics we expose for Daml services supporting a Prometheus metrics reporter.
+The following sections contain the list of common metrics we expose for Daml services supporting a Prometheus metrics reporter.
```
Comment (da-katmurp) [inline docs/source/ops/common-metrics.rst:7]:
```suggestion
The following sections contain the common metrics we expose for Daml services supporting a Prometheus metrics reporter.
```

File: docs/source/ops/common-metrics.rst [hunk 2]
```diff
@@ -207,3 +207,22 @@ daml_executor_tasks_queue_remaining
 - **Description**: Additional elements that this queue can ideally accept without blocking.
 - **Type**: Gauge
 - **Observation**: Only available for `type` = `thread_pool`
+
+Pruning metrics
```
Comment (da-katmurp) [inline docs/source/ops/common-metrics.rst:211]:
```suggestion
Pruning Metrics
```
(could be a capital, I'm never too sure .. better to check context from other docs )

File: docs/source/ops/common-metrics.rst [hunk 3]
```diff
@@ -207,3 +207,22 @@ daml_executor_tasks_queue_remaining
...
+***************
+
+We expose the following metrics for all pruning processes. These metrics have the following common labels attached:
```
Comment (da-katmurp) [inline docs/source/ops/common-metrics.rst:214]:
```suggestion
We expose the following metrics for all pruning processes. These metrics have the following labels:
```

**Review by da-katmurp — APPROVED** [review]
just another few wee things :)

**Review by nicu-da — APPROVED** [review]

**Pull request merged**